tasks/kernel_matrix_testing/README.md

```diff
@@ -0,0 +1,231 @@
+# Kernel Matrix Testing system
+
+## Overview
+The Kernel Matrix Testing system is a new approach for testing system-probe. It uses libvirt and qemu to launch pre-provisioned VMs over a range of distributions and kernel versions. These VMs are used for running the test suite of system-probe.  
+
+Developers can check out this confluence page for more details about the system.   
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement. -->
**🥜 nitpick**
The confluence page link is missing

Reply:
So is the confluence page itself :sweat_smile: